tools-local/unix/acquire-tool.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+# Downloads the declared version of the specified tool.
```

Comment:
I kind of feel the usage method should be sufficient explanation of the scripts.  Comments like this just adds another place that is likely to get out of date over time.